sql_generators/glean_usage/templates/dau_reporting_clients_first_seen_v1.query.sql

```diff
@@ -0,0 +1,63 @@
+{{ header }}
+
+WITH
+  _current AS (
+    SELECT
+      client_info.client_id,
+      -- TODO: Once this field exists we should update the reference.
+      CAST(NULL AS STRING) AS usage_profile_id,
+      {% raw %}
+      {% if is_init() %}
+      DATE(MIN(submission_timestamp)) AS first_seen_date,
+      {% else %}
+      @submission_date AS first_seen_date,
+      {% endif %}
+      {% endraw %}
+    FROM
+      `{{ dau_reporting_stable_table }}`
+    WHERE
+      client_info.client_id IS NOT NULL
+      {% raw %}
+      {% if is_init() %}
+      AND DATE(submission_timestamp) > "2014-10-10"
+      {% else %}
+      AND DATE(submission_timestamp) = @submission_date
+      {% endif %}
+      {% endraw %}
+    GROUP BY
+      client_id
+  ),
+_previous AS (
+  SELECT
+    client_id,
+    usage_profile_id,
+  FROM
+    `{{ dau_reporting_clients_first_seen_table }}`
+  WHERE
+    {% raw %}
+    {% if is_init() %}
+    False
+    {% else %}
+    first_seen_date < @submission_date
+    {% endif %}
+    {% endraw %}
+)
+
+SELECT
+  first_seen_date,
+  client_id,
+  usage_profile_id,
+FROM
+  _previous
+LEFT JOIN
+  _current
```

Comment:
```suggestion
 FROM
  _current
LEFT JOIN
  _previous
```
@kik-kik shouldn't the first_seen be selected from `_current` if its missing in the `_previous`?

Reply:
updated.